{
  "description": "An issue was discovered in py-lmdb 0.97. There is a divide-by-zero error in the function mdb_env_open2 if mdb_env_read_header obtains a zero value for a certain size field.",
  "releases": {
    "bullseye": {
      "repositories": {
        "bullseye": "0.86-1.1"
      },
      "status": "undetermined",
      "urgency": "medium**"
    },
    "sid": {
      "repositories": {
        "sid": "0.86-1.1"
      },
      "status": "undetermined",
      "urgency": "medium**"
    },
    "stretch": {
      "repositories": {
        "stretch": "0.86-1"
      },
      "status": "undetermined",
      "urgency": "medium**"
    }
  },
  "scope": "remote"
}